webpack.config.js

```diff
@@ -1,12 +1,13 @@
-var webpack = require('webpack');
-var config = {};
+const webpack = require('webpack');
+const config = {};
 
 function generateConfig(name) {
-  var uglify = name.indexOf('min') > -1;
-  var config = {
+  const mode = name.includes('min') ? 'production' : 'development';
+  return {
+    mode,
     entry: './index.js',
     output: {
-      path: 'dist/',
+      path: `${__dirname}/dist`,
```

Comment:
absolute path is required for webpack 4 now.

per the docs, `__dirname` seems appropriate 

https://nodejs.org/docs/latest/api/modules.html#modules_dirname